core/src/main/java/com/linecorp/armeria/internal/HttpHeaderUtil.java

```diff
@@ -14,25 +14,20 @@
  * under the License.
  */
 
-package com.linecorp.armeria.client;
+package com.linecorp.armeria.internal;
```

Comment:
Could we move this to `armeria.internal.client`?

Reply:
Moved to `armeria.internal.client` package.